book.rb

```diff
@@ -0,0 +1,17 @@
+require_relative 'items'
+
+class Book < Item
+  attr_accessor :archived
+
+  def initialize(publisher, cover_state)
+    super
+    @publisher = publisher
+    @cover_state = cover_state
+  end
+
+  private
+
+  def can_be_archived?
+    archived || @cover_state == 'bad'
```

Comment:
- [ ] According to the project requirements, the Book class should return true if parent's method returns true OR if cover_state equals to "bad", Can you please try to replace **archived || @cover_state == 'bad'** with **super || @cover_state == 'bad'**.

Reply:
Okay